resfit/fit_resonator/fit_S_data.py

```diff
@@ -559,10 +592,12 @@ def PlotFit(x,
         resonance = 1/(1 + params[1] +1j*params[3])
     else:
         resonance = 1 + 1j*0
-    ax.plot(np.real(resonance),np.imag(resonance),'*',color = 'red',label = 'resonance',markersize = 7)
-    ax3.plot(params[2],np.log10(np.abs(resonance))*20,'*',color = 'red',label = 'resonance',markersize = 7)
-    ax4.plot(params[2],np.angle(resonance),'*',color = 'red',label = 'resonance',markersize = 7)
-
+    ax.plot(np.real(resonance),np.imag(resonance),'*',color = 'red',label =
+            'resonance',markersize=18)
```

Comment:
I would make the red lines a bit thinner so we can see the blue circle markers behind it more easily.

Reply:
I think the red lines are thin enough. The highlighted line controls the size of the star marking the point of the resonance on the circle and magnitude / phase plots. Here is an example of what the code generates with the settings as above. I think the red lines are thin enough to see the open blue circles underneath.
[Uploading circle_fit_1SP_InP_220416.pdf…]()